### PR TITLE
✨ option to hide views from tab row

### PR DESCRIPTION
**Pull request by iantrich**

### CODE REVIEW

**Comment by iantrich** [comment]
@balloob any objections to this going in for 0.91 after I resolve the conflict?

**Comment by balloob** [comment]
Anything merged before next week Wednesday will be in 0.91

File: src/panels/lovelace/hui-root.ts [hunk 1]
```diff
@@ -248,47 +259,49 @@ class HUIRoot extends LitElement {
...
-                                  @click="${this._moveViewRight}"
-                                  ?disabled="${(this._curView! as number) +
-                                    1 ===
-                                    this.lovelace!.config.views.length}"
-                                ></ha-paper-icon-button-arrow-next>
-                              `
-                            : ""}
-                        </paper-tab>
-                      `
+                    ${this.lovelace!.config.views.map((view) =>
```
Comment (balloob) [inline src/panels/lovelace/hui-root.ts]: This code is getting out of control. Let's push it before the return statement.

```ts
const tabs = [];

for (const view of this.lovelace!.config.views) {
  if (view.hidden) { continue; }
  const icon = view.icon ? html`
    <ha-icon…>
  ` : undefined;
  }
  tabs.push(html`<paper-tab>${icon}…`)
```

**Comment by balloob** [comment]
We should do this with a user filter instead.

**Pull request closed without merging**